docs/starter-kit-apis/passports/passports_overview.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 1
+---
+
+# Starter Kit Passports Overview
```

Comment:
There seems to be mixed use of the words "Passports" and "Passport" in both file names and content. Since, the standard is called GA4GH Passport. Let's use "Passport" everywhere.